src/react-aria-modal.js

```diff
@@ -142,7 +134,7 @@ class Modal extends React.Component {
 
     if (props.underlayStyle) {
       for (const key in props.underlayStyle) {
-        if (!props.underlayStyle.hasOwnProperty(key)) continue;
+        if (!props.underlayStyle.includes(key)) continue;
```

Comment:
AFAIK `underlayStyle` is an object, and does not have the `includes` prototype method. Could you explain this a little bit more?

Reply:
Ahhh, this was me trying to get around a test error without fully thinking it through. I'll revert this change, @ArfatSalman